sleap/gui/overlays/base.py

```diff
@@ -23,20 +24,47 @@
 
 @attr.s(auto_attribs=True)
 class BaseOverlay(abc.ABC):
-    """
-    Abstract base class for overlays.
-
+    """Abstract base class for overlays.
     Most overlays need rely on the `Labels` from which to get data and need the
-    `QtVideoPlayer` to which a `QGraphicsObject` item will be added, so these
+    `QtVideoPlayer` to which a `QGraphicsItem` will be added, so these
     attributes are included in the base class.
+    Args:
+        labels: the `Labels` from which to get data
+        player: the `QtVideoPlayer` to which a `QGraphicsObject` item will be added
+        items: stores all `QGraphicsItem`s currently added to the player from this
+            overlay
```

Comment:
For future, if you would like code from a different branch, then a `git pull origin <branch-name>` is best (otherwise these changes to the same files are registered as "merge conflicts" even if they are the same lines of code).